src/pandapipes/pf/result_extraction.py

```diff
@@ -57,12 +57,12 @@ def extract_all_results(net, calculation_mode):
 def get_basic_branch_results(net, branch_pit, node_pit):
     from_nodes = branch_pit[:, FROM_NODE].astype(np.int32)
     to_nodes = branch_pit[:, TO_NODE].astype(np.int32)
-
+    fluid = get_fluid(net)
     t0 = node_pit[from_nodes, TINIT_NODE]
     t1 = node_pit[to_nodes, TINIT_NODE]
-    mf = branch_pit[:, LOAD_VEC_NODES]
-    vf = mf / get_fluid(net).get_density((t0 + t1) / 2)
-    return branch_pit[:, VINIT], mf, vf, from_nodes, to_nodes, t0, t1, branch_pit[:, RE], \
+    vf = branch_pit[:, MINIT] / fluid.get_density((t0 + t1) / 2)
```

Comment:
Why don't we use the specific function for branches here? -> `get_branch_density`